Big-Spender/readme.md

```diff
@@ -112,7 +112,7 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+select date as Month, SUM(amount) as total_spent from spends where date between '2021-03-01' and '2021-04-01' group by date;
```

Comment:
How could we get the date month? is the date actually the 'month'? 
we are using between here, but what if we had a date like 2021-04-20? what would happen with our query then?